index.js

```diff
@@ -18,6 +20,7 @@ var defaultContentDisposition = staticValue(null)
 var defaultStorageClass = staticValue('STANDARD')
 var defaultSSE = staticValue(null)
 var defaultSSEKMS = staticValue(null)
+var currentSize = 0
```

Comment:
it actually does - i moved the functions around sufficiently that its old location meant it got defined below where it was used, so the lint complained